.github/workflows/publish_imgs.yml

```diff
@@ -70,18 +70,33 @@ jobs:
     # Utilizing nixos/nix docker image v2.3
     container:
       image: nixos/nix@sha256:af330838e838cedea2355e7ca267280fc9dd68615888f4e20972ec51beb101d8
-    
+
     steps:
       # Checks out repository to docker container
       # Utilizes the commit from the open PR, which was gotten from the previous step
       # Using SHA v2
-      - name: Pulls Repository Code
+      - name: Pulls repository code
         uses: actions/checkout@28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b
         with:
           ref: ${{ needs.verify.outputs.sha }}
 
+      - name: Install bash
+        run: |
+          mkdir /github/home/.nix-defexpr
+          ln -s /nix/var/nix/profiles/default /github/home/.nix-profile
+          ln -s /nix/var/nix/profiles/per-user/root/channels /github/home/.nix-defexpr/channels
+          nix-env -iA nixpkgs.bash
+        shell: sh {0}
+        continue-on-error: false
```

Comment:
Cachix installation required Bash, but it's not provided in the `nixos` image. Interestingly, the workflow runs as root, and root in the image *does* have a `nix-env` setup ready to go, but GH Actions insists on a separate "home" folder. So this basically recreates the `.nix-profile` and `.nix-defexpr` needed to get the `nix-env` install to work.

The `sh {0}` is necessary because `sh` in the image is actually Busybox's version which doesn't support the `-e` implied command that GH Actions would want to add.

Earlier, I tried forking the `cachix-action` repo myself, made the change that I *thought* would circumvent `bash` in favor of `sh`. Didn't work as I expected. 💩

Reply:
Ya I think this is just a consequence of using that container which may be something we need to reevaluate how we are doing breaking down our steps/jobs. But Im goood with this for time being!